core/src/main/java/org/jruby/RubyInstanceConfig.java

```diff
@@ -1852,6 +1852,9 @@ public boolean shouldPrecompileAll() {
     public static boolean IR_COMPILER_DEBUG = Options.IR_COMPILER_DEBUG.load();
     public static boolean IR_WRITING = Options.IR_WRITING.load();
     public static boolean IR_READING = Options.IR_READING.load();
+    public static boolean IR_INLINER = Options.IR_INLINER.load();
+    public static int IR_INLINER_THRESHOLD = Options.IR_INLINER_THRESHOLD.load();
+    public static boolean IR_INLINER_VERBOSE = Options.IR_INLINER_VERBOSE.load();
```

Comment:
You could just move these closer to where you use them...I think we want to get away from the pattern of static fields on RubyInstanceConfig for every little thing.

Reply:
@headius ok I will change this quick...although not entirely sure where I will put them maybe in IRManager